app/views/shared/stats/facts/_coverage_stats.html.erb

```diff
@@ -101,10 +101,16 @@
       countries due to difference in methodologies and datasets used to assess
       protected area coverage and differences in the base maps used to measure
       terrestrial and marine area of a country or territory.
+      
       <% if @country && @country.iso_3 == 'JPN' %>
         Download <a href="https://www.cbd.int/doc/world/jp/jp-nr-05-en.pdf" class="link-with-icon">Japan’s fifth national report (pdf).</a>
       <% end %>
     </p>
+    <% if @country && has_restricted_sites? %>
```

Comment:
As mentioned above, if you include the `@country &&` bit in the helper function it would be better